---
string id_eeg_signal
int32 size_experiment